lms/djangoapps/class_dashboard/dashboard_data.py

```diff
@@ -624,11 +673,12 @@ def get_students_problem_grades(request, csv=False):
 
         header = [_("Name").encode('utf-8'), _("Username").encode('utf-8'), _("Grade").encode('utf-8'), _("Percent").encode('utf-8')]
         for student in students:
-
-            percent = 0
-            if student['max_grade'] > 0:
-                percent = round(student['grade'] * 100 / student['max_grade'])
-            results.append([student['student__profile__name'], student['student__username'], student['grade'], percent])
+            this_user_id = student['student__id']
+            if this_user_id not in non_student_id_list:
```

Comment:
Could we just say if student['student__id'] not in non_student_list for conciseness?